### ISSUE TITLE
Views FileSystem

### ISSUE THREAD

**Issue by cjango** [kataras/iris#1615]
When install ,something wrong

# github.com/kataras/iris/view
../../../go/pkg/mod/github.com/kataras/iris@v0.0.2/view/blocks.go:76:37: undefined: blocks.AssetFunc
../../../go/pkg/mod/github.com/kataras/iris@v0.0.2/view/blocks.go:76:66: undefined: blocks.AssetNamesFunc
../../../go/pkg/mod/github.com/kataras/iris@v0.0.2/view/blocks.go:77:10: s.Engine.Assets undefined (type *blocks.Blocks has no field or method Assets)

**Comment by kataras** [comment]
Hello @cjango you are using the `master` version. Some days ago I pushed a big commit which removed all `Binary` methods from the views and instead all views now accept an `http.FileSystem` (and a string as they used to) as correctly requested by an Iris user . The examples and the changelog (HISTORY.md file) were also updated. So for your case, here is the [blocks embedded example](https://github.com/kataras/iris/blob/master/_examples/view/template_blocks_1_embedded/main.go).

**Quick fix for you**:

Open a terminal:
```sh
$ go get -u github.com/go-bindata/go-bindata/...
$ go-bindata -fs -prefix ".views" ./views/...
```

Replace the `Blocks(...).Binary` with the following:
```go
app.RegisterView(iris.Blocks(AssetFile(), ".html"))
```

You are ready.
